### PR TITLE
Add missing team_id argument to conversationList API method

### PR DESCRIPTION
**Pull request by RiverVanRain**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jolicode/slack-php-api",
+    "name": "rivervanrain/slack-php-api",
```
Comment (lyrixx) [inline composer.json:2]: This is wrong

**Review by lyrixx — COMMENTED** [review]
Thanks for your first contribution.

Did you know the code was generated? Or did you update the code manually?

**Comment by damienalexandre** [comment]
Thanks for your contribution. We have a documentation about how to update the SDK: https://github.com/jolicode/slack-php-api/blob/main/docs/4-updating-sdk.md and as @lyrixx said, we cannot accept this PR as it is.

**Comment by mpyw** [comment]
@damienalexandre @lyrixx @RiverVanRain It looks that the official API spec seems to be broken 😢  https://github.com/jolicode/slack-php-api/issues/137#issue-1106501528

**Pull request closed without merging**